rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/util/PackageNameConverter.groovy

```diff
@@ -82,6 +87,9 @@ class PackageNameConverter {
           release = parts.pop()
           version = parts.pop()
           name = parts.join("-")
+          qualifiedName = "${name}${versionDelimiter}${version}-${release}"
```

Comment:
Since `qualifiedName` is composed the same way for both package types, maybe just make it a method on `OsPackageName`? It can still return `null` if `release` is missing.